test_cli.py

```diff
@@ -386,7 +388,7 @@ class TestModule(object):
     a decorator to register new modules and a class method to run modules by name.
     """
 
-    modules = OrderedDict()
+    modules: Dict[str, Any] = OrderedDict()
```

Comment:
We should try to avoid `Any` as much as we can. I think `Any` is the same as not specifying a type. Should it be `TestModule` instead?

Reply:
Are we using python 3.7 or newer? If not we can't refer to the class within itself. I worked out another way.

Reply:
You can use a string: `"TestModule"`

Reply:
Yeah, I did it in my latest update.